crates/holochain_p2p/src/op_store.rs

```diff
@@ -92,36 +92,21 @@ impl HolochainOpStore {
 impl OpStore for HolochainOpStore {
     fn process_incoming_ops(&self, op_list: Vec<Bytes>) -> BoxFut<'_, K2Result<Vec<OpId>>> {
         Box::pin(async move {
-            let dht_ops = op_list
-                .into_iter()
-                // Filter to make casting the size below safe
-                .filter(|op| op.len() <= u32::MAX as usize)
-                .map(|op| {
-                    Ok((
-                        op.len() as u32,
-                        decode::<_, DhtOp>(op.as_ref())
-                            .map_err(|e| K2Error::other_src("Could not decode op", e))?,
-                    ))
-                })
-                .collect::<K2Result<Vec<(u32, DhtOp)>>>()?;
-
-            let ids = dht_ops
-                .iter()
-                .map(|(_, op)| {
-                    let op_hashed = DhtOpHashed::from_content_sync(op.clone());
-                    op_hashed.hash.to_k2_op()
-                })
-                .collect();
+            let mut dht_ops = Vec::with_capacity(op_list.len());
```

Comment:
Simplifying the function here. The op len is no longer needed.

Reply:
I still think we should reject ops that are clearly too large here and not store them but then that crates a funky situation where we still want to return the hash to remove it at the caller.. A problem for another time!